Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake

```diff
@@ -151,6 +151,7 @@ function(cgal_arr_2_add_target exe_name source_file)
     set(name ${exe_name}_${suffix})
   endif()
   add_executable(${name} ${source_file})
+  target_link_libraries(${name} CGAL::CGAL)
```

Comment:
I think you need to link with `CGAL::CGAL_Core` as well, because the `CMakeLists.txt` used:
```cmake
find_package(CGAL REQUIRED COMPONENTS Core)

include(${CGAL_USE_FILE})
```